TF_Service_and_Tooling_Unity/Assets/TouchFree/Tooling/Scripts/Connection/MessageReceiver.cs

```diff
@@ -132,14 +132,19 @@ void HandleConfigState(ConfigState _configState)
             }
         }
 
+        Vector2 lastKnownCursorPosition = new Vector2();
```

Comment:
Reminder to make sure changes made to the copy of this file in TF_Application are also made to the tooling